<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>751</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Asynchronous</string>
    </property>
    <widget class="QLabel" name="label_AsyBeginType">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>26</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Begin-type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_AsyBeginProps">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Begin-props</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_AsyFinishType">
     <property name="geometry">
      <rect>
       <x>381</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Finish-type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_AsyFinishProps">
     <property name="geometry">
      <rect>
       <x>379</x>
       <y>130</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Finish-props</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="label_AsyFinishPropsR">
     <property name="geometry">
      <rect>
       <x>454</x>
       <y>130</y>
       <width>281</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="label_AsyBeginTypeR">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="label_AsyBeginPropsR">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="label_AsyFinishTypeR">
     <property name="geometry">
      <rect>
       <x>454</x>
       <y>20</y>
       <width>281</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_DeviceCheck">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Check device</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>861</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_UnregisterProcesser">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>4</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Unregister  video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_RegisterProcesser">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>4</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Register video</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>26</y>
       <width>481</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Video</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>14</y>
        <width>461</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_Cam1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Cam0">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_markerCountR">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>num</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_frameRate">
     <property name="geometry">
      <rect>
       <x>505</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Framerate:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Refresh">
     <property name="geometry">
      <rect>
       <x>501</x>
       <y>72</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_frameRateR">
     <property name="geometry">
      <rect>
       <x>579</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>num</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_scandist">
     <property name="geometry">
      <rect>
       <x>505</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Distance:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_scandistR">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>11</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>num</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_triangleCount">
     <property name="geometry">
      <rect>
       <x>621</x>
       <y>33</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Triangle:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_noMarkerDetected">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>14</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>noMarkerDetected</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pointCount">
     <property name="geometry">
      <rect>
       <x>621</x>
       <y>13</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Points:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_tooFlat">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>34</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>tooFlat</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_trackLost">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>54</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>trackLost</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_markerCount">
     <property name="geometry">
      <rect>
       <x>505</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Markers:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pointCountR">
     <property name="geometry">
      <rect>
       <x>696</x>
       <y>13</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>num</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_scanStatus">
     <property name="geometry">
      <rect>
       <x>623</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_scanStatusR">
     <property name="geometry">
      <rect>
       <x>667</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>status</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_triangleCountR">
     <property name="geometry">
      <rect>
       <x>698</x>
       <y>33</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>num</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>100</y>
       <width>351</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Scan</string>
     </property>
     <widget class="QPushButton" name="pushButton_ScanExitScan">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>12</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>12</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ST_RAPID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ScaneEnterScan">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>12</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_scanExportFile">
      <property name="geometry">
       <rect>
        <x>106</x>
        <y>127</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>sketchfab</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>solidedge</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_scanMesh">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>96</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Mesh</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ScanExportFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>126</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Export file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_scanSave">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>96</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Pause">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>72</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Pre">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>14</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_start">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>43</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_scanCancelScan">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>134</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_scanEndScan">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>102</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>End </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_OpenProject">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>63</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ScanOpenProject">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>62</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Open project</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ScanNewProject">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>37</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>New project</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ScanLastSimplifyParams">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>97</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Last simplify params</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_scanSimplify">
      <property name="geometry">
       <rect>
        <x>192</x>
        <y>127</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Simplify</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>8</y>
      <width>96</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Heartbeat:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="intValue" stdset="0">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFixScan">
    <property name="title">
     <string>Rapid scan</string>
    </property>
   </widget>
   <addaction name="menuFixScan"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
